### DISCUSSION TITLE
Rotate the player pawn towards camera's linetrace detected element

### DISCUSSION THREAD

**Discussion by oserrat** [Sixze/ALS-Refactored#155] [Q&A]
Hello, basically I want to modify the spine rotation so it always point to centre of the screen while aiming, so I think it should take into account the current weapon position and rotation, but I don't know how to implement this.

**Comment by colorindarkness** [comment]
You would use GetLookAtRotation to get the rotation, and you would perform that off a location on the weapon. Then you can set the rotation.
